ext/ZygoteExt.jl

```diff
@@ -0,0 +1,97 @@
+module ZygoteExt
+
+using Zygote: pullback
+using ZygoteRules: @adjoint
+using SciMLBase: ODESolution, issymbollike, sym_to_index, remake, getobserved
+
+@adjoint function getindex(VA::ODESolution, i::Int, j::Int)
```

Comment:
why does this need an overload?

Reply:
Added a comment about this, without this there is a method ambiguity.